REST client/trainings/src/main/java/training/evaluation/training/model/Training.java

```diff
@@ -1,8 +1,11 @@
 package training.evaluation.training.model;
 
+import org.bson.types.Binary;
 import org.springframework.data.annotation.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
 
+import java.util.List;
+
 @Document(collection = "trainings")
 public class Training {
```

Comment:
To avoid writing getters and setters you can take a look at [lombok](https://projectlombok.org/features/all)